base/applications/mspaint/main.cpp

```diff
@@ -250,16 +249,13 @@ _tWinMain (HINSTANCE hThisInstance, HINSTANCE hPrevInstance, LPTSTR lpszArgument
     if (registrySettings.ShowStatusBar)
         ShowWindow(hStatusBar, SW_SHOWNOACTIVATE);
 
-    RECT scrlClientWindowPos = {0, 0, 0 + 500, 0 + 500};
-    scrlClientWindow.Create(scrollboxWindow.m_hWnd, scrlClientWindowPos, NULL, WS_CHILD | WS_VISIBLE);
+    /* creating the window inside the scroll box, on which the image in hDrawingDC's bitmap is drawn */
```

Comment:
Recommendation: use CPP commenting syntax // instead of /* */

Reply:
OK.